### PR TITLE
Add a directory for released datasets, and initial smart contract audit results

### PR DESCRIPTION
**Pull request by agroce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 * [Podcasts](#podcasts)
 * [Security Reviews](#security-reviews)
 * [Workshops](#workshops)
+* [Datasets](#datasets)
```
Comment (dguido) [inline README.md]: Please sort by name

File: README.md [hunk 2]
```diff
@@ -198,3 +199,9 @@
 | [Smart Contract Security Automation Workshop](workshops/Smart%20Contract%20Security%20Automation%20-%20ETHBerlin%202018) | ETH Berlin 2018 | September 2018 |
 | [Manticore EVM Workshop](workshops/Manticore%20-%20EthCC%202018) | EthCC 2018 | March 2018 |
 | [Manticore Workshop](workshops/Manticore%20-%20GreHack%202017) | GreHack 2017 | October 2017 |
+
+## Datasets
```
Comment (dguido) [inline README.md]: Please move above Podcasts to sort by name

File: datasets/smart_contract_audit_findings/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Smart Contract Audit Findings
+
+Format of findings files is CSV containing:
```
Comment (dguido) [inline datasets/smart_contract_audit_findings/README.md]: Maybe: "Audit findings are presented in a CSV with the following fields:"

File: README.md (hunk 1 above)
Comment (agroce) [reply README.md]: Done

File: datasets/smart_contract_audit_findings/README.md (hunk 3 above)
Comment (agroce) [reply datasets/smart_contract_audit_findings/README.md]: Done

File: README.md (hunk 2 above)
Comment (agroce) [reply README.md]: Done

**Pull request merged**